core/silkworm/state/intra_block_state.cpp

```diff
@@ -255,7 +258,7 @@ void IntraBlockState::set_storage(const evmc::address& address, const evmc::byte
     if (prev == value) {
         return;
     }
-    storage_[address][key].current = value;
+    storage_[address].current[key] = value;
     journal_.push_back(std::make_unique<state::StorageChangeDelta>(address, key, prev));
```

Comment:
This causes a contructor of `unique_ptr` and then a move into vector.
Why not simply ...
```suggestion
   journal_.emplace_back(new state::StorageChangeDelta(address, key, prev));
```

Reply:
Done